### PR TITLE
Theme Framework for Drawing App

### PR DESCRIPTION
**Pull request by brosscode**
Adds a basic theme setup for the Drawing App. Additionally, changed the color of the Canvas to be white instead of the original Light Grey Color.

### CODE REVIEW

File: app/src/main/res/values-night/styles.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="DrawingTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <item name="backgroundColor">@color/black</item>
+        <item name="android:background">@color/black</item>
```
Comment (ForzenAcademy) [inline app/src/main/res/values-night/styles.xml]: Make a background attribute `attr` also

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="DrawingTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <item name="backgroundColor">@color/arsenic</item>
+        <item name="android:background">@color/arsenic</item>
```
Comment (ForzenAcademy) [inline app/src/main/res/values/styles.xml]: You aren't declaring your attr for sheet color in either of your themes

File: app/src/main/res/values-night/styles.xml (hunk 1 above)
Comment (brosscode) [reply app/src/main/res/values-night/styles.xml]: made into attr

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (brosscode) [reply app/src/main/res/values/styles.xml]: technically my sheet isn't in this PR so I removed this and will include it in an appropriate PR

**Review by ForzenAcademy — APPROVED** [review]

**Pull request merged**